rules/windows/process_creation/win_file_permission_modifications.yml

```diff
@@ -19,14 +19,21 @@ detection:
       CommandLine|contains: '/grant'
     - Image|endswith: '\attrib.exe'
       CommandLine|contains: '-r'
-  condition: selection
+  filter:
+    - CommandLine: 
+        - 'C:\Windows\system32\cmd.exe /C ICACLS C:\ProgramData\dynatrace\gateway\config\config.properties /grant :r *S-1-5-19:F'
```

Comment:
Does that `*` character really appear in the command line or do you want to use a wildcard. 
If it's the latter, please use `CommandLine|contains|all` instead and split that string into two.